src/test/scala/uk/gov/hmrc/agentmtdidentifiers/model/UtrSpec.scala

```diff
@@ -22,12 +22,23 @@ class UtrSpec extends FlatSpec with Matchers {
 
   it should "be true for a valid UTR" in {
     Utr.isValid("2000000000") shouldBe true
+    Utr.isValid("9000000001") shouldBe true
+    Utr.isValid("7000000002") shouldBe true
+    Utr.isValid("5000000003") shouldBe true
   }
 
   it should "be false when it has more than 10 digits" in {
     Utr.isValid("20000000000") shouldBe false
   }
 
+  it should "be false when it is empty" in {
+    Utr.isValid("") shouldBe false
+  }
+
+  it should "be false when it is all zeros" in {
```

Comment:
I agree too in terms of test coverage. I was following the cases in the Jira. @sarthak83?